storage/testonly/admin_storage_tester.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/google/trillian"
 	spb "github.com/google/trillian/crypto/sigpb"
 	"github.com/google/trillian/storage"
+	testonly2 "github.com/google/trillian/testonly"
```

Comment:
nit: Could we find a better alias? `trilliantestonly`, `trltestonly`, `ttestonly`, something else?